docs/articles/testkube-dashboard-workflows-templates.md

```diff
@@ -1,102 +1,102 @@
 # Workflow Templates
+ 
+**Workflow Templates** are reusable, standardized configuration instructions for running tests specific to a range of tools such as Playwright and Selenium. With a standard set of instructions 
```

Comment:
".. are resuable configuration blocks for your Workflows that can be used to make sure your Workflows follow specific patterns or best practices, for example when using tools like Playwright or Selenium. With a standard.. "